client/editSchedule.js

```diff
@@ -110,10 +129,10 @@ setupCompetitionCalendar = function(template, $calendarDiv, $editModal) {
         $editModal.modal('show');
       },
       eventDrop: function(calEvent, delta, revertFunc, jsEvent, ui, view) { // Existing entry dragged
-        eventChanged(calEvent);
+        eventChanged(calEvent, revertFunc);
```

Comment:
Interesting. So previously, we allowed events to be dragged off the schedule, but now you're making that impossible?

What about the cases where a multiday competition becomes loses a day with events, or the start/end of a competition changes? I should probably just try this out, but I'm guessing that the multifield custom validation you added in https://github.com/cubing/ccm/pull/250/files#diff-bb1d087a5188cd88ac9aee475765c592R56 isn't going to prevent these things from happening.